### PR TITLE
Release Debian CitusHA 2.0.0

### PR DESCRIPTION
**Pull request by thanodnl**

### CODE REVIEW

**Comment by thanodnl** [comment]
Test run of release branch
```
$ citus_package -p=debian/jessie -p=debian/stretch -p=ubuntu/xenial -p=ubuntu/trusty local release 2>&1 | tee -a citus_package.log

...

$ grep -Ei '(warning|\bi|\be|\bw):' citus_package.log | sort | uniq -c
   4 W: postgresql-11-citus-ha-2.0: binary-without-manpage usr/bin/citusha
   4 W: postgresql-11-citus-ha-2.0: debian-changelog-line-too-long line 7
   4 dpkg-buildpackage: warning: using a gain-root-command while being root
```

```
$ citus_package -p=debian/jessie -p=debian/stretch -p=ubuntu/xenial -p=ubuntu/trusty local nightly 2>&1 | tee -a citus_package.log

...

$ grep -Ei '(warning|\bi|\be|\bw):' citus_package.log | sort | uniq -c
   1 W: postgresql-11-citus-ha-dbgsym: package-has-long-file-name 74 (83) > 80
   4 W: postgresql-11-citus-ha: binary-without-manpage usr/bin/citusha
   4 dch warning: Previous package version was Debian native whilst new version is not
   4 dpkg-buildpackage: warning: using a gain-root-command while being root
```

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,24 @@
+citus-ha (2.0.0) testing; urgency=low
+
+  * Rework of the CLI to `citusha <verb> <object>`
+
+  * CLI can be used to find the uri of the monitor and the formations
+
+  * Added support for a proxy port (eg. the pgbouncer port fronting the database)
+
+  * Added support for a cutsom listen address that postgres will listen on
```
Comment (lfittl) [inline debian/changelog]: "cutsom" => "custom"

File: pkgvars [hunk 2]
```diff
@@ -1,6 +1,6 @@
 pkgname=citus-ha
 hubproj=citus-ha
 pkgdesc='Citus HA'
-pkglatest=1.0.0
-releasepg=11
+pkglatest=2.0.0
+releasepg=10,11
```
Comment (lfittl) [inline pkgvars]: Are you sure this works? (package build fails for me unless I change this to `releasepg=11`)

**Review by lfittl — CHANGES_REQUESTED** [review]

**Comment by lfittl** [comment]
Successfully tested for Postgres 11 on Ubuntu Bionic - so as long as we can resolve the build failures, this should be good to go.

File: pkgvars (hunk 2 above)
Comment (thanodnl) [reply pkgvars]: I was hoping this worked, as we do it in the build scripts of citus as well.
`10` and `11` build separately, but apparently not together.

I think we want to release both versions of citusha, maybe @velioglu can shed some light on how to proceed here.

**Review by lfittl — APPROVED** [review]

**Comment by thanodnl** [comment]
We will release the ubuntu/debian packages for pg 11 today, more work is needed to get the cross product to work between 10 and 11.

From a quick chat with @velioglu it is unclear if we can just revert the commit that enabled pg11 to get the pg10 build (although those are currently in package cloud).

I will spend more time verifying pg10 on ubuntu contains all the binaries one needs to run. And get the build to deploy both citusha for pg10 and pg11

**Pull request merged**